src/theme.js

```diff
@@ -1,3 +1,12 @@
 const { default: CreateMuiTheme } = require('@material-ui/core/styles/createMuiTheme');
 
-module.exports = CreateMuiTheme({});
+module.exports = CreateMuiTheme({
+    palette: {
+        duckYellow: {
```

Comment:
I wonder if the example of theming should be something more likely to not need changing/removing when starting a project.  Otherwise all of our projects will inevitably end-up with `duckYellow` in the theme :P

Reply:
Yeah, that sounds good! Do you have any suggestions?

Reply:
Here are two thoughts:

1. Pull the color from MUI: https://material-ui.com/customization/color/#color
2. Make the secondary text color yellow for the application, rather than adding a new branch of the theme.  This will organically be changed for each application.  Would also address the testing issue from the other comment, if that's a real issue.

Reply:
These are some strong ideas, and I decided to implement both to show usage https://github.com/BigRoomStudios/strangeluv/pull/237/commits/1a54f9cfc90fd4672c742e31f0fbbea2e0762c1f

I'm using the `secondary` color in the `HomePage` component, and pulled in `amber` to use `amber[500]` as the secondary color.